### ISSUE TITLE
authorize method doesn't work

### ISSUE THREAD

**Issue by DavoCg** [alkawryk/tinderjs#20]
Hi, I have a valid fbToken and fbId get with Tinderbot module but the authorize method of tinderjs doesn't seem to work

``` javascript
this.authorize = function(fbToken, fbId, callback) {

    console.log('fbToken : ',fbToken)
    console.log('fbId : ',fbId)

    tinderPost('auth',
      {
        facebook_token: fbToken,
        facebook_id: fbId
      },
      function(error, res, body) {

        console.log('error : ', error);
        console.log('body  : ', body);



        if (!error && body.token) {
          xAuthToken = body.token;
          _this.userId = body.user._id;
          callback();
        } else if (body.error){
          throw "Failed to authenticate: " + body.error
        }
      });
  };
```

The console.log('body  : ', body);  returns : 

``` javascript
 body  { status: 'access denied' } 
```

My question is : Is this still possible to use tinderjs module to use the Tinder API ? and why i've got this access denied ? 

Thanks guys

**Comment by monkeymanbob** [comment]
I'm working through this too. I think the reason it's failing is because there's a facebook token request by post that never returns. I don't know what's going on with it but if I figure it out I'll let you know.

**Comment by unenglishable** [comment]
This is still broken.  I'm going to take a look at it too.

**Comment by mobyg** [comment]
any updates on this?

**Comment by yfujimot** [comment]
I haven't found any tinder library that works anymore. Any suggestions?

**Comment by unenglishable** [comment]
@garryhicks:  I think this thing package is pretty much dead.

@yfujimot:  You'll have to reverse engineer a new one.  Other than that, you can try getting the facebook auth to work for this package; it might just be using something outdated.

**Comment by mobyg** [comment]
I think you're right.

**Comment by yfujimot** [comment]
It's actually very easy to make it work!

**Comment by kelpycreek** [comment]
HI! This library is working fine for me, (although the documentation could use some help).

To fix the authentication error, follow the directions in this thread:
https://github.com/alkawryk/tinderjs/issues/16

**Comment by yfujimot** [comment]
Yup, I can confirm this works as well! If you send a ton of requests too quickly though, it'll return an error so watch out for that.

**Comment by sarment0** [comment]
@kelpycreek @yfujimot  This workaround is not working anymore. Any clue?  (Tested on Mac OSX)

**Comment by mayeaux** [comment]
Just published a library that will let you get your Facebook token and id programatically :) https://github.com/tinderjs/tinderauth
